packages/ui/src/pages/MyStudiesPage/components/StudyComponent.vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+defineProps<{ name: string; description: string }>();
+</script>
+
+<template>
+  <div class="flex flex-row border-b border-neutral-300 w-3/4 h-24">
+    <div class="flex flex-row my-20 justify-left w-4/5 self-center">
```

Comment:
Same as above, this flex row inside a flex row is a red flag, also `flex-row` is the default behavior for flex anyway so you never need it :)